smoke-tests/build.gradle.kts

```diff
@@ -37,6 +37,7 @@ tasks {
       "jboss" to listOf("**/JBossEapSmokeTest.*"),
       "jetty" to listOf("**/JettySmokeTest.*"),
       "liberty" to listOf("**/LibertySmokeTest.*"),
+      "profiler" to listOf("**/Profiler*"),
```

Comment:
Eek, was this just not running before? 😬

Reply:
It was -- the profiler tests were in a different module (`:testing:profiler-tests`). I put this one into `:smoke-tests` to avoid copy pasting lots of things; and I'm thinking of refactoring the other profiler tests and putting them here too.